build/ci/evergreen.yml

```diff
@@ -722,7 +722,7 @@ tasks:
   ################################################################
   # Unstable Publish
   # RPM Distros
-  - name: push_rhel70_enterprise
```

Comment:
The name here confused me a bit, as it looks like you push to both 62 and 70 repos?

Reply:
yes, we are, I moved 70 lower to keep the versions sorted